aws/naming_resources.md

```diff
@@ -0,0 +1,18 @@
+# Naming Resources
+
+## IAM Resources
+
+AWS operator needs a set of roles in customer accounts to be able to operator against the AWS API. Most of the customers need bounded policy to be security complaint. Here we define a convention to help them to achieve that and a same time have a coherent way to name the roles.
```

Comment:
The text is still not wrapped. Let me know when you need help with that.

Reply:
Hmm according to Visual code it is 
<img width="777" alt="Screen Shot 2020-04-08 at 16 38 05" src="https://user-images.githubusercontent.com/892157/78796972-5e856d00-79b7-11ea-8c1d-0a0ea06ffbad.png">

Reply:
done